redis/src/connection.rs

```diff
@@ -222,6 +229,13 @@ where
     }
 }
 
+/// URL format: `{redis|rediss}://[<username>][:<password>@]<hostname>[:port][/<db>]`
+///
+/// - Basic: `redis://127.0.0.1:6379`
+/// - Username & Password: `redis://user:password@127.0.0.1:6379`
+/// - Specifying DB: `redis://127.0.0.1:6379/database`
```

Comment:
It may be good to add a password-only example too in each spot, e.g.

```
/// - Password only: `redis://:password@127.0.0.1:6379`

```

This actually comes up a lot, so maybe it will help someone in the future.